lib/is-core.js

```diff
@@ -0,0 +1,5 @@
+var core = require('./core');
+
+module.exports = function isCore(x) {
+    return Object.prototype.hasOwnProperty.call(core, x);
```

Comment:
unfortunately this introduced a bug into v1.13.0 (see #231), by changing this from a "truthy" check, to a "presence" check - and the "core" object relies on `true`/`false` to differentiate between a core module in _some_ version of node, versus a core module in the _current_ version of node. This was inadvertently fixed when I extracted this logic into https://npmjs.com/is-core-module.